cdc/cdc.hh

```diff
@@ -166,6 +166,21 @@ struct db_context final {
 /// param[in] schema schema of a table for which CDC tables are being created
 seastar::future<> setup(db_context ctx, schema_ptr schema);
 
+// cdc log table operation
+enum class operation : int8_t {
+    // note: these values will eventually be read by a third party, probably not privvy to this
+    // enum decl, so don't change the constant values. 
+    pre_image = 0, update = 1, row_delete = 2, range_delete_start = 3, range_delete_end = 4, partition_delete = 5
+};
+
+// cdc log data column operation
+enum class column_op : int8_t {
+    // same as above
```

Comment:
better to reference operation enum by name to make it safe in case when someone inserts another code in between.

Reply:
What do you mean "by name"? And what do you mean "inserts another code inbetween"?

Reply:
I mean something like this "This class is restricted the same way as enum class operation".
Inserting another code inbetween:
enum class operation {
// important comment
...
}

class foo {
// not important comment
...
}

enm class column_op {
// same as above
}